resources/views/layouts/modal.blade.php

```diff
@@ -16,7 +16,7 @@
                       enctype="multipart/form-data"
                       data-controller="layouts--form"
                       data-action="layouts--form#submit"
-                      data-layouts--form-button-animate="#screen-modal-{{$key}}"
+                      data-layouts--form-button-animate="#submit-modal-{{$key}}"
```

Comment:
We are sending another form:
```php
  <form class="modal-content"
    id="screen-modal-form-{{$key}}"
    method="post"
```
What is this change for?

Reply:
for animating the submit button when clicked..

when a submit button is clicked, in the submit action/method of layouts/form_controller.js you call the function ` this.animateButton();` 

`animateButton() {
        const button = this.data.get('button-animate');
        const text = this.data.get('button-text') || '';`

Reply:
I understand the animation. I want to say that the forms with the name `#submit-modal-{{$key}}`  does not exist

Reply:
oh ok.
 So here this `data-layouts--form-button-animate="#submit-modal-{{$key}}"` is not referring to a form but rather storing the attribute or value `button-animate` in the stimulusjs  controller. 

Please [check here](https://stimulusjs.org/reference/data-maps.html) for reference

You are getting those attributes or controller data in this line from the file `form_controller.js`

`const button = this.data.get('button-animate');
 const text = this.data.get('button-text') || '';`

Reply:
Yes, that’s right, it’s my fault that I tried to mislead you for myself.